app/src/common/chromium/com/igalia/wolvic/browser/api/impl/TextInputImpl.java

```diff
@@ -13,7 +13,7 @@
 import com.igalia.wolvic.browser.api.WTextInput;
 
 public class TextInputImpl implements WTextInput {
-    SessionImpl mSession;
+    WSession mSession;
```

Comment:
Why this? I mean, at this level we can refer to the SessionImpl if we want, just curious

Reply:
Oops. This is my mistake and redundant fix. Removed in the latest PR.